pipelines/pipeline_katello_upgrade.yml

```diff
@@ -41,51 +45,57 @@
 - hosts: pipeline-upgrade-centos7
```

Comment:
This step is now repeated a few times. I wonder if it makes sense to factor it out to a role you you can easily apply that a few times.

Reply:
good idea